modules/@apostrophecms/schema/index.js

```diff
@@ -1534,7 +1534,7 @@ module.exports = {
         let errors = [];
 
         for (const field of schema) {
-          if (field.readOnly) {
+          if (field.disabled) {
```

Comment:
Now the official name of this property. No one objected to my decision to embrace the term that's used all over the frontend components, so I'll open a documentation ticket.

Reply:
I didn't realize this was what you were suggesting for that. I don't think this should change. It still makes sense when writing a schema to designate things as `readOnly`. When that is converted to UI, that UI is `disabled` because that's the language of inputs.

Reply:
There are way more occurrences of `field.disabled` on the front end than there were of `field.readOnly` on the back end, and it's a reasonable choice, so I went along with it. But if you prefer `field.readOnly` I'm inclined to keep that if Stu and Ben don't have any big concerns (we did talk about this today). I will ask.